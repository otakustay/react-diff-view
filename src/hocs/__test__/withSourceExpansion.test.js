import renderer from 'react-test-renderer';
import {withSourceExpansion} from '..';

const ComponentIn = () => <div />;

describe('withSourceExpansion', () => {
    test('returns a hoc function', () => {
        expect(typeof withSourceExpansion()).toBe('function');
    });

    test('hoc function return a component', () => {
        expect(typeof withSourceExpansion()(ComponentIn)).toBe('function');
    });

    test('render component', () => {
        const ComponentOut = withSourceExpansion()(ComponentIn);
        expect(renderer.create(<ComponentOut />).toJSON()).toMatchSnapshot();
    });
});
