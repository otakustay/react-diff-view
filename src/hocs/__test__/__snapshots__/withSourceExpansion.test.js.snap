// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withSourceExpansion render component 1`] = `<div />`;
